config/hera.yaml

```diff
@@ -1,74 +1,82 @@
-machine: hera
+--- 
 account: nems
-queue: batch
-partition: hera
-scheduler: slurm
+branch: 
+  - develop
+  - jedwards/pio_update2
+compiler: 
+  - intel
+  - gfortran
+  - pgi
 corespernode: 40
-compiler: [intel, gfortran, pgi]
-branch: [develop, jedwards/pio_update2]
-nuopcbranch: develop
-git-https: True
-intel:
-  versions:
-    18.0.4:
-      compiler: intel/18.0.5.274
-      netcdf: netcdf-hdf5parallel/4.7.4
-      mpi:
-        intelmpi:
-          module: impi/2018.4.274
-          pythontest: True
-          mpi_env_vars:
-            var1: ESMF_MPIRUN=mpirun.srun
-        mpiuni:
-          module: None
-      extra_commands:
-        cmd1: "tar xvfz ~/pytest-input.tar.gz"
-    extramodule: cmake
-gfortran:
+gfortran: 
   test_time: "2:00:00"
-  versions:
-    9.2.0:
+  versions: 
+    ? "9.2.0"
+    : 
       compiler: gnu/9.2.0
-      netcdf: netcdf/4.7.2
-      hdf5: hdf5/1.10.5
-      mpi:
-        openmpi:
-          module: openmpi/3.1.4
-          pythontest: True
-          mpi_env_vars:
-            var1: ESMF_MPIRUN=mpirun.srun
-        mpiuni:
-          module: None
-      extra_env_vars:
+      extra_commands: 
+        cmd1: "tar xvfz ~/pytest-input.tar.gz"
+      extra_env_vars: 
         var1: ESMF_NETCDF=split
         var2: ESMF_NETCDF_INCLUDE=$NETCDF/include
         var3: ESMF_NETCDF_LIBPATH=$NETCDF/lib
-        var4: ESMF_NETCDF_LIBS="-lnetcdff -lnetcdf -lhdf5_hl -lhdf5 $HDF5ExtraLibs"
+        var4: "ESMF_NETCDF_LIBS=\"-lnetcdff -lnetcdf -lhdf5_hl -lhdf5 $HDF5ExtraLibs\""
         var5: ESMF_NETCDF=nc-config
-      extra_commands:
-        cmd1: "tar xvfz ~/pytest-input.tar.gz"
-    9.2.0b:
+      hdf5: hdf5/1.10.5
+      mpi: 
+        mpiuni: 
+          module: None
+        openmpi: 
+          module: openmpi/3.1.4
+          mpi_env_vars: 
+            var1: ESMF_MPIRUN=mpirun.srun
+          pythontest: true
+      netcdf: netcdf/4.7.2
```

Comment:
This netcdf version won't work - can you please change it?

Reply:
@jedwards4b updated.

https://github.com/esmf-org/esmf-test-scripts/commit/27acffca80694ecec057a24591bfb3ba4cf18de9